sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -6,27 +6,29 @@
 
 /// <reference types="node" />
 
```

Comment:
What would be the updated version?
Are we going to pass this as a minor version update?

Reply:
bump it to 7.6.0-beta.1.  I plan to ship 7.5.1 for a bug fix.